### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by notlfish**
Responsive web design clone of https://perma.cc/M5ZV-Q2D6.
In this branch, the cloning of the page is to be completed; continuing from the work done on branch milestone-1.

The stylings should be responsive for smartphones (0-767px width), tablets (768px-1023px width) and desktop (> 1024px width).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,693 @@
...
+                                2 hour ago
+                                <span class="highlight-shares-small"><i class="fa fa-share-alt" aria-hidden="true"></i>
+                                    5</span>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section id="latest-news">
+                <h2 class="section-title">Latest news ></h2>
```
Comment (sinansevgi) [inline index.html]: - [x] Title of the section header should be a link.
```suggestion
                <h2 class="section-title"><a href="#">Latest news></a></h2>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,693 @@
...
+                            <h5 class="news-title">Signal is drama-free for now, but it should prepare for the worst</h5>
+                            <ul class="credits-list">
+                                <li><a href="#" class="normal-link">Avhimanshu Ghoshal</a></li>
+                                <li class="point-separator">&middot;</li>
+                                <li>8 hours ago</li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </section>
```
Comment (sinansevgi) [inline index.html]: - [x] All the article images and titles should be a link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,693 @@
...
+                            <li class="cat-extra-info"><a href="#" class="normal-link"><span class="orange-bullet">&bull;</span>How Netflix shapes mainstream culture, explained by data </a></li>
+                            <li class="cat-extra-info"><a href="#" class="normal-link"><span class="orange-bullet">&bull;</span>Christmas is better with Tom Vasel’s YouTube board game reviews</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </section>
+            <section id="latest-deals">
+                <div class="latest-deals-header">
+                    <h2 class="section-title">Latest deals ></h2>
+                    <p class="section-options">See all deals</p>
```
Comment (sinansevgi) [inline index.html]: - [x] 'See all details'  should be a link.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,693 @@
...
+                            <li class="cat-extra-info"><a href="#" class="normal-link"><span class="orange-bullet">&bull;</span>The podcasts that got us through 2020</a></li>
+                            <li class="cat-extra-info"><a href="#" class="normal-link"><span class="orange-bullet">&bull;</span>How Netflix shapes mainstream culture, explained by data </a></li>
+                            <li class="cat-extra-info"><a href="#" class="normal-link"><span class="orange-bullet">&bull;</span>Christmas is better with Tom Vasel’s YouTube board game reviews</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </section>
+            <section id="latest-deals">
+                <div class="latest-deals-header">
+                    <h2 class="section-title">Latest deals ></h2>
```
Comment (sinansevgi) [inline index.html]: - [x] Title of the section should be a link.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,693 @@
...
+                        <div class="discount mobile-hidden"><p>87% OFF</p></div>
+                        <img alt="Familio" src="assets/deals-img-8.jpg" class="deals-image">
+                        <div class="deals-text">
+                            <h5 class="deals-title"> Familio Private Backup Platform: Lifetime Subscription</h5>
+                            <p class="deals-old-price">$300</p>
+                            <p class="deals-new-price">$39.99</p>
+                        </div>
+                    </div>
+                </div>
+            </section>
```
Comment (sinansevgi) [inline index.html]: - [x] All the article images and titles should be a link

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 
-----

Hello team,
You’re doing a great job. Your project is almost done. There are some small things you should fix before you continue with your journey :wave: 

Kindly check the issues down below

-----

Please also be aware of styling may change when you wrap each of these elements with the <a> tag. So you may want to create classes.

-----

## "As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project."

-----
You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

File: index.html [hunk 6]
```diff
@@ -0,0 +1,690 @@
...
+                            <div id="bot-nav-links">
+                                <div class="bot-nav-text-links">
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Latest</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Hard fork</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Plugged</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Readme</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Growth quarters</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link over-1215">Shift</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link over-1215">Neural</a></div>
+                                </div>
```
Comment (Berabjesus) [inline index.html:47]: - [x] **[Optional]** Your project does not have a medium: 768px breakpoint, which could help to fix the small mess you have here.

Original at 781px 👇 
image

yours at 781px 👇 
image

**Review by Berabjesus — APPROVED** [review]
# **Status: PROJECT APPROVED** 
#### Excellent job on this project @DanSam5K  and @notlfish . :fireworks: Keep it up.  :champagne: :champagne: **All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete. 👍** 
- You have fixed the issues raised in the previous review.
- The cloned page looks as it should and meets all the project specifications
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)

### **Before you merge to the main branch please have a look at the following OPTIONAL comments.**

- [x] [Optional] Avoid duplicate usage of CSS rules, instead create one class for a rule and use it on the HTML. (For example, `display: flex` is used 19 times in the CSS) instead create a class for it, for example, `d-flex` and use that class in the HTML.

I wish you all the very best in your next tasks:1st_place_medal:
#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

File: index.html (hunk 5 above)
Comment (DanSam5K) [reply index.html]: Noted sir!

File: index.html (hunk 6 above)
Comment (DanSam5K) [reply index.html:47]: Noted and thank you for your kind response! will fix the bug. 👍

**Pull request merged**